lms/static/scripts/postmessage_json_rpc/server.js

```diff
@@ -39,7 +39,7 @@
  *     const server = new Server(['https://hypothes.is']);
  *     server.register('requestConfig', requestConfig);
  */
-export default class Server {
+export class Server {
```

Comment:
Making this a named export rather than a `default` export was more convenient for importing the class as a type at one point during development of this PR. It is less important in the final version, but it still has a benefit since it ensures that the symbol has the same name everywhere. There is a wider conversation to be had about our conventions for exports generally.